examples/integrations/oci/vault-cdi/src/main/java/io/helidon/examples/integrations/oci/vault/cdi/VaultResource.java

```diff
@@ -108,31 +133,40 @@ public String decrypt(@PathParam("text") String cipherText) {
     @GET
     @Path("/sign/{text}")
     public String sign(@PathParam("text") String dataToSign) {
-        return vault.sign(Sign.Request.builder()
-                                  .keyId(signatureKeyOcid)
-                                  .algorithm(Sign.Request.ALGORITHM_SHA_224_RSA_PKCS_PSS)
-                                  .message(Base64Value.create(dataToSign)))
-                .signature()
-                .toBase64();
+        return crypto.sign(SignRequest.builder()
+                                   .signDataDetails(SignDataDetails.builder()
+                                                            .keyId(signatureKeyOcid)
+                                                            .signingAlgorithm(SignDataDetails.SigningAlgorithm.Sha224RsaPkcsPss)
+                                                            .message(Base64Value.create(dataToSign).toBase64())
+                                                            .build())
+                                   .build())
+                .getSignedData()
+                .getSignature();
     }
 
     /**
-     * Verify a signature.
+     * Verify a signature. The base64 encoded signature is the entity
      *
      * @param dataToVerify data that was signed
-     * @param signature signature text
+     * @param signature    signature text
      * @return whether the signature is valid or not
      */
-    @GET
-    @Path("/sign/{text}/{signature: .*}")
+    @POST
```

Comment:
(Should the return type of `verify` be `void`, then? Or maybe just `Response` and then do whatever you need to? `String` seems odd to me but maybe I'm just not thinking clearly.)